data/ka'het ships.txt

```diff
@@ -0,0 +1,149 @@
+	outfits
+		"Hyperdrive"
+		"Ka'het Small Cooling" 
+		"Ka'het Large Cooling"
+		"Ka'het Blaster Turret" 2
+		"Ka'het Laser Turret" 2
+		"Ka'het Laser" 2
+		"Ka'het Shield" 2
+
+	engine -20 66 0.6
+	engine -26.5 66 0.8
+	engine 20 66 0.6
+	engine 26.5 66 0.8
+	gun -9 -74 "Ka'het Laser"
+	gun 9 -74 "Ka'het Laser"
+	turret -13.5 -29 "Ka'het Blaster Turret"
+	turret -14 -12.5 "Ka'het Laser Turret"
+	turret 13.5 -29 "Ka'het Blaster Turret"
+	turret 14 -12.5 "Ka'het Laser Turret"
+	explode "tiny explosion" 5
+	explode "small explosion" 20
+	explode "medium explosion" 25
+	explode "large explosion" 15
+	"final explode" "final explosion medium"
+	description "The Maeri is the smallest Het still created by the Builders' stations, a prime example of the genetic technology they once possessed. Its small size is compensated by the fact that this specie of Het can effectively cloak, hiding its location from most types of sensors."
+
+ship "Telis'het"
+	plural "Telis'heta"
+	sprite "ship/telis'het"
+	"never disabled"
+	attributes
+		category "Heavy Warship"
+		"cost" 32600000
+		"shields" 47000
+		"hull" 26300
+		"required crew" 1
+		"bunks" 1
+		"mass" 620
+		"drag" 6.9
+		"heat dissipation" .35
+		"fuel capacity" 800
+		"ramscoop" 0.82
+		"cargo space" 71
+		"outfit space" 583
+		"weapon capacity" 264
+		"energy capacity" 20000
+		"energy generation" 32.3
+		"heat generation" 40
+		"hull repair" 0.5
+		"shield energy" 0.3
+		"shield heat" 9.2
+		"thrust" 62.8
+		"thrusting energy" 11.4
+		"thrusting heat" 3.5
+		"flare sprite" "effect/remnant flare/small"
+			"frame rate" 5
+		"flare sprite" "effect/remnant flare/medium"
+			"frame rate" 4
+		"flare sound" "plasma medium"
+		"turn" 2031.7
+		"turning energy" 6.38
+		"turning heat" 2.9
+		weapon
+			"blast radius" 300
+			"shield damage" 1200
+			"hull damage" 600
+			"hit force" 1800
+	outfits
+		"Hyperdrive"
+		"Ka'het Large Cooling" 2
+		"Ka'het Shield" 7
+		"Ka'het Blaster Turret" 2
+		"Ka'het EMP Launcher" 2
+		"Ka'het Laser Turret" 3
+		"EMP Torpedo" 32
+		"Ka'het Blaster" 2
+		
+	engine -27 75.5 0.6
+	engine -39.5 82.5 0.8
+	engine 27 75.5 0.6
+	engine 39.5 82.5 0.8
+	gun -11.5 -94 "Ka'het EMP Launcher"
+	gun -23.5 -87.5 "Ka'het Blaster"
+	gun 11.5 -94 "Ka'het EMP Launcher"
+	gun 23.5 -87.5 "Ka'het Blaster"
+	turret -27 -41 "Ka'het Blaster Turret"
+	turret -41.5 -50.5 "Ka'het Laser Turret"
+	turret 0 -53 "Ka'het Laser Turret"
+	turret 27 -41 "Ka'het Blaster Turret"
+	turret 41.5 -50.5 "Ka'het Laser Turret"
+	explode "small explosion" 25
+	explode "medium explosion" 30
+	explode "large explosion" 20
+	explode "huge explosion" 5
+	"final explode" "final explosion large"
+	description "When the Builders were designing the exoskeleton of their Telis, they came to the conclusion that it would have been impossible to make it strong enough without compromising its agility and the Het's integrity. Instead they focused heavily on improving their shield tech, while also giving it many of their regenerators."
```

Comment:
description "When the Builders designed the exoskeleton of the Telis, they came to the conclusion that it would have been impossible to make it resilient enough without compromising its agility and integrity. Instead, they focused heavily on improving their shield technology, giving the Telis many of its regenerators."